src/backend/distributed/sql/udfs/citus_prepare_pg_upgrade/latest.sql

```diff
@@ -67,6 +69,22 @@ BEGIN
        objects.colocationid
     FROM citus.pg_dist_object objects,
          pg_catalog.pg_identify_object_as_address(objects.classid, objects.objid, objects.objsubid) address;
+
+    SELECT count(*) > 0
```

Comment:
I think the question is: When does `citus_depends_on_citus_stat_activity` ever becomes false?

Reply:
We call this function again to check if it is idempotent

Reply:
let's add this as a comment as well, thanks

Reply:
ah, already added